starlette/context.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any, Optional
+
+try:
+    from contextvars import Context, ContextVar, copy_context
+except ImportError:  # pragma: no cover
+    Context = object  # type: ignore
+    ContextVar = copy_context = None  # type: ignore
```

Comment:
This module can be cleaned up better. I was trying to encapsulate all of the `try: import contexvars` checks into one module